.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+  - "2.7"
```

Comment:
I'm also developing this on 3.5, should we stick to that for testing? I don't see anything in this project that should prevent it.